tcms/kiwi_auth/tests/test_views.py

```diff
@@ -329,6 +365,16 @@ def test_get_template_names(self):
 class TestPasswordResetView(TestCase):
     """Test for password reset view"""
 
-    def test_form_class(self):
-        response = self.client.get(reverse("tcms-password_reset"))
-        self.assertIsInstance(response.context["form"], forms.PasswordResetForm)
+    def setUp(self):
+        self.password_reset_url = reverse("tcms-password_reset")
+
+    def test_open_password_reset_page(self):
+        response = self.client.get(self.password_reset_url)
+
+        self.assertContains(response, ">Password reset</button>")
+
+    def test_current_site_password_reset(self):
+        data = {"email": "tester@example.com"}
+        response = self.client.post(self.password_reset_url, data)
```

Comment:
use `follow=True` and assert that the password reset page didn't error out.

Ideally mock the email sending function and assert that a password reset email was sent to the user.

Reply:
I use `follow=True` and assert if 200 response comes or not. Also, the form is inherited from Django forms, therefore, do we actually need to mock the email sending function?

Reply:
> do we actually need to mock the email sending function

Yes because you have no other way of checking that this function was triggered. That's why the rest of the auth tests are doing the same.